operator/config/crd/bases/forklift.konveyor.io_migrations.yaml

```diff
@@ -293,6 +293,9 @@ spec:
                       description: The VM Namespace Only relevant for an openshift
                         source.
                       type: string
+                    ovfConfig:
```

Comment:
@bennyz is less happy after seeing this change :)
there are two similar things:
1. returning the OVF from the conversion pod - there are arguments in favor or and opposing to returning the entire OVF (already discussed as part of this PR). I'm in favor of returning the entire OVF as this PR currently does
2. storing the OVF in the status - I'm ok with storing only the firmware here, and possibly add more fields from the OVF when needed, as @bennyz suggested

Reply:
changed, now we save only the firmware on the controller side.